cart.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+
+<html lang=" en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Harshna Haswani">
+  <meta name="description" content="Ecommerce site for chocolates, cookies, and more">
+  <title> Melting Moments — Cart</title>
+  <link rel="icon" type="image/png" href="/images/favicon.ico">
+  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png">
+  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
+  <link rel="manifest" href="/site.webmanifest">
+
+  <link rel="stylesheet" href="/styles/main.css">
+
+</head>
+
+<body>
+
+
+  
+  <nav class="nav nav-site" role="main">
+    <a class="logo" href="/">
+      <img src="/images/apple-touch-icon.png"></img>
+    </a>
+    <h2><a class="heading list-heading" href="/">Melting Moments</a></h2>
+    <div class="form-item status-plain-outline">
+      <input type="search" name="search" id="search" class="form-input" placeholder="search" aria-label="search"
+        autocomplete="name">
+    </div>
+    <ul class="list list-inline">
+
+
+      <li class="list-item">
+        <a href="/cart.html">Cart</a>
+      </li>
+
+
+
+
+
```

Comment:
Do look out at these empty spaces in case you forgot to remove.

Reply:
Thank you for the PR! I'll remove them